internal/aws/cwlogs/utils.go

```diff
@@ -41,7 +41,7 @@ func ValidateRetentionValue(input int64) error {
 
 // Check if the tags input is valid
 func ValidateTagsInput(input map[string]*string) error {
-	if input != nil && len(input) < 1 {
```

Comment:
```
=== FAIL: . TestLoadConfig/awscloudwatchlogs/e2-no-retries-short-queue (re-run 1) (0.00s)
    config_test.go:106: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter/config_test.go:106
        	Error:      	Received unexpected error:
        	            	invalid amount of items. Please input at least 1 tag or remove the tag field
        	Test:       	TestLoadConfig/awscloudwatchlogs/e2-no-retries-short-queue

=== FAIL: . TestLoadConfig (re-run 1) (0.00s)
```
Looks like this change broke tests

Reply:
Good catch, in this case `nil` is valid. Reverted